src/pages/ProductPage.tsx

```
@@ -0,0 +1,258 @@
+                  className="w-full h-full object-cover"
+                />
+              </div>
+
+              {gallery.length > 1 && (
+                <div className="mt-4 grid grid-cols-4 gap-3">
+                  {gallery.map((img, index) => (
+                    <button
+                      key={`${img.src}-${index}`}
+                      onClick={() => setActiveImage(img.src)}
+                      className={`rounded-lg overflow-hidden border ${activeImage === img.src ? 'border-primary' : 'border-white/10'}`}
+                    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)